image-matrix.json

```diff
@@ -1,4 +1,6 @@
  [
-    {"runtimeVersion": "8u302-slim", "latest": false},
-    {"runtimeVersion": "11.0.12-slim", "latest": true}
+    {"runtimeVersion": "8u302-jre-slim", "latestTags": ["jre-8"]},
```

Comment:
I think it might be clearer if the property `latestTags` was renamed `tags`, as the tag may not include `latest`.

Reply:
Complete in 3ed623365e0fef384fcc94b088254cce9b9cbe17